pkg/controller/clientutil/kubernetes_test.go

```diff
@@ -122,7 +122,7 @@ func TestKubeClientUpdate(t *testing.T) {
 	client := clientutil.NewKubeClient(cl)
 
 	updatedCluster := cluster.DeepCopy()
-	updatedCluster.Spec.KubernetesVersion = anywherev1.Kube126
+	updatedCluster.Spec.KubernetesVersion = anywherev1.Kube127
```

Comment:
nit: Don't think we care about unit tests while deprecating a K8s version right. Like we have so many unit tests still referencing very old K8s version. But this shouldn't affect anything.